community/modules/database/slurm-cloudsql-federation/variables.tf

```diff
@@ -73,3 +73,9 @@ variable "network_id" {
     error_message = "The network id must be provided in the following format: projects/<project_id>/global/networks/<network_name>."
   }
 }
+
+variable "private_vpc_connection_peering" {
+  description = "The name of the VPC Network peering connection, used only as dependency for Cloud SQL creation."
+  type        = any
```

Comment:
I think this is a better type constraint.

```hcl
  type = string
  default = ""
```

Seems OK to be nullable given that it's unused in a real sense.

Reply:
I think the depends_on does not care about the value so much as whether it is populated. But if you've tested with `null` we can leave it as-is.